voluncheer/templates/chatroom/room.html

```diff
@@ -0,0 +1,49 @@
+<!-- placeholder room file -->
+
+{% load static %}
+
+<!DOCTYPE html>
```

Comment:
Same comment as the chat_homepage.html file.

Reply:
Same, it interferes with the demo interface. So I suggest we make the change later.

Reply:
Same here, will get it done as soon as this is passed to the frontend